requirements/extras.txt

```diff
@@ -1 +1 @@
-cvxopt==1.2.4
+cvxopt==1.2.4
```

Comment:
This line appears to have only whitespace changes. It seems that the line could be omitted from the changeset.